### PR TITLE
add debug hints

### PR DESCRIPTION
**Pull request by tbg**
[Review on Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/2935)

### CODE REVIEW

File: acceptance/localcluster/localcluster.go [hunk 1]
```diff
@@ -369,7 +369,13 @@ func (l *Cluster) startNode(i int) *Container {
 	c.Name = node(i)
 	uri := fmt.Sprintf("https://%s", c.Addr(""))
 	// Infof doesn't take positional parameters, hence the Sprintf.
-	log.Infof(fmt.Sprintf("started node:\n\tname:  %s\n\tui:    %[2]s\n\ttrace: %[2]s/debug/requests\n\tpprof: %[2]s/debug/pprof/profile\n\tlogs:  %s\n\tcerts:  %s", c.Name, uri, localLogDir, l.CertsDir))
+	log.Infof(fmt.Sprintf(`*** started %s ***
```
Comment (tamird) [inline acceptance/localcluster/localcluster.go]: doesn't really matter, but `%[1]s`?

File: acceptance/localcluster/localcluster.go [hunk 2]
```diff
@@ -369,7 +369,13 @@ func (l *Cluster) startNode(i int) *Container {
...
+  ui:    %[2]s
+  trace: %[2]s/debug/requests
```
Comment (tamird) [inline acceptance/localcluster/localcluster.go]: don't you need the port here?

**Comment by tamird** [comment]
LGTM modulo nits

File: acceptance/localcluster/localcluster.go (hunk 2 above)
Comment (tschottdorf) [inline acceptance/localcluster/localcluster.go]: Addr() already gives a host:port pair.

File: acceptance/localcluster/localcluster.go (hunk 1 above)
Comment (tschottdorf) [inline acceptance/localcluster/localcluster.go]: sure, why not.

**Pull request merged**